hapi21/routes/pathTraversalSemanticFileSecurityBypass/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
This route is throwing an internal server error (both here and in Hapi20). It seems some .ejs file is missing or sth like that